graph-db/src/utils/indexing/indexCmrCollection.js

```diff
@@ -0,0 +1,38 @@
+const { indexRelatedUrl } = require('./indexRelatedUrl');
+
+exports.indexCmrCollection = async (collection, gremlin) => {
+  const { meta, umm } = collection;
+  const conceptId = meta['concept-id'];
+  const { EntryTitle: entryTitle, DOI: doi, RelatedUrls: relatedUrls } = umm;
+  const hasDOI = !!doi.DOI;
+  let doiUrl = 'Not supplied';
```

Comment:
CMR uses 'Not provided' for when things are.. not provided. Is this value getting stored in graphDB? Seems like we'd want ensure we're adding consistent data into this db, or write null/empty values instead of a string.